OtripleS.Web.Api/Models/Guardian/Guardian.cs

```diff
@@ -0,0 +1,24 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OtripleS.Web.Api.Models.Guardian
+{
+    public class Guardian : IAuditable
+    {
+        public Guid Id { get; set; }
+        public string FirstName { get; set; }
+        public string FamilyName { get; set; }
+        public GuardianRelationship Relationship { get; set; }
+        public DateTimeOffset CreatedDate { get; set; }
+        public DateTimeOffset UpdatedDate { get; set; }
+        public Guid CreatedBy { get; set; }
+        public Guid UpdatedBy { get; set; }
```

Comment:
This is not the correct model we talked about here: https://github.com/hassanhabib/OtripleS/issues/144#issuecomment-688039223

Reply:
Should I used this one. 

public class Guardian : IAuditable {

    public Guid Id {get; set;}
    public string FirstName {get; set;}
    public string FamilyName {get; set;}
    public DateTimeOffset CreatedDate {get; set;}
    public DateTimeOffset UpdatedDate {get; set;}
    public Guid CreatedBy {get; set;}
    public Guid UpdatedBy {get; set;}
}

Reply:
Added Correct model in this Commit **Last Changes in Guardian Model**